src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/missingjavadoctype/Example1.java

```diff
@@ -0,0 +1,17 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="MissingJavadocType"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.missingjavadoctype;
+
+// xdoc section -- start
+class Example1 {
+  public class testClass1 {}
```

Comment:
Please add above:
```
/** Javadoc.*/
public class testClass0 {}
```

To show what is expected code, and it will never have violation.

Reply:
Done